specs/ff-asn-selected-filter-cloud/ff-asn-selected-filter-cloud.md

```diff
@@ -1,86 +1,212 @@
-# ff-asn-selected-filter-cloud
-The `ff-asn-selected-filter-cloud` displays all active filters using `ff-asn-group-element`'s.
+# ff-filter-cloud
+The `ff-filter-cloud` displays all filters that are currently selected through `ff-asn`.
 
 ## Usage
+
+Minimum setup:
+
 ```html
-<ff-asn-selected-filter-cloud>
-    <ff-asn-group-element>
-        <span slot="selected">{{name}}</span>
-        <span slot="unselected">{{name}}</span>
-    </ff-asn-group-element>
-</ff-asn-selected-filter-cloud>
+<ff-filter-cloud>
+    <div data-filter-container>
+        <span data-filter>{{name}}</span>  <!-- this element is repeated inside the filter-container -->
+    </div>
+</ff-filter-cloud>
+```
 
-<!-- OR -->
+With additional custom HTML:
 
-<ff-asn-selected-filter-cloud>
-    <section>
-        <!-- Will be rendered inside this section -->
-        <ff-asn-group-element>
-            <span slot="selected">{{name}}</span>
-            <span slot="unselected">{{name}}</span>
-        </ff-asn-group-element>
-    </section>
-</ff-asn-selected-filter-cloud>
+```html
+<ff-filter-cloud>
+    <h4>The Filter Cloud</h4>
+
+    <div data-filter-container>
```

Comment:
please remove the `data-filter-container` attribute. It's actually unnecessary.

Reply:
How would you decide where to put the filter elements?
Putting them directly into `ff-filter-cloud` prevents adding further HTML after the elements.
```html
<ff-filter-cloud>
    <h4>The Filter Cloud</h4>
    <div data-filter-container></div>
    <h4>Footer</h4>
</ff-filter-cloud>
```